### DISCUSSION TITLE
Elasticsearch ES v8.x planned?

### DISCUSSION THREAD

**Discussion by tomiles** [broadinstitute/seqr#3238] [General]
Is there any work planned to support ES v8.x ? since this version removes support for mapping types **[1]**.
ES 7 already gave deprecation warnings but it still worked.  So supporting data import to an ES 8 cluster does not seems as painless as with previous upgrades. Since the data model seems to still rely on types.

[1] https://www.elastic.co/guide/en/elasticsearch/reference/8.6/removal-of-types.html#removal-of-types

**Comment by hanars** [comment]
We are currently working to deprecate elasticsearch in seqr entirely and move to a different search backend, so we do not have any plans to update

**Reply by tomiles** [reply]
Thanks for the quick answer, in that case we will limit our cluster updates to ES v7.x.x
Looking forward to learning more about the new search backend.

**Reply by SimonLi5601** [reply]
We are looking forward to the new backend as well. Is there any repo we can follow to learn about how the new loading pipeline looks like? Thanks!

**Reply by hanars** [reply]
I will update once there is anything that is publicly available, but there will be no new repo, all updates will be released in the main seqr repo
